report/renderer/category-renderer.js

```diff
@@ -317,6 +317,10 @@ export class CategoryRenderer {
    * @return {DocumentFragment}
    */
   renderScoreGauge(category, groupDefinitions) { // eslint-disable-line no-unused-vars
+    if (category.displayMode === 'ratio') {
```

Comment:
this seems like a bit of a misleading path where we've asked for a gauge but receive a ratio

new method `renderCategoryScore` that does this delegation like you've done in the flow report?

Reply:
I started doing this but backed out once I reached
https://github.com/GoogleChrome/lighthouse/blob/ee53af31fca149eeede3b9ac217d2b5882a58782/report/renderer/report-renderer.js#L167

We would just keep that line the same right?

Reply:
My preference would be to replace this line with `renderCategoryScore === renderCategoryScore` and rename the PWA one to renderCategoryScore. In fact that name makes more sense anyway because it doesn't have a gauge :)

We can alias `renderScoreGauge` to `renderCategoryScore` in PWA category for backcompat until v9 in case any partners are using it.